woocommerce/facebook-for-woocommerce
# Generate PHP7.0 compatible code.

### Changes proposed in this Pull Request:

Explicit scope quantifier on const is not supported in PHP<7.1. Since we required public and public is implicit removing the `const` keyword is all that is needed to fix #1906.

Closes #1906.

### How to test the changes in this Pull Request:



1. Edit a product.
2. Open the Facebook tab.
3. Categories setting should work
![image](https://user-images.githubusercontent.com/17271089/116370508-c39e4180-a80a-11eb-8710-9a862d50ea37.png)


### Changelog entry


 * Fix - PHP<7.1 incompatible code for Google Taxonomy Setting in products.
